tikv-client/src/main/java/com/pingcap/tikv/operation/KVErrorHandler.java

```diff
@@ -127,6 +140,18 @@ public void handle(RespT resp) {
         throw new StatusRuntimeException(Status.fromCode(Status.Code.UNAVAILABLE).withDescription(error.toString()));
       }
     }
+
+    // Other error handling logic
+    // Currently we need to handle potential other errors from coprocessor responses.
+    if (getOtherError != null) {
+      String otherError = getOtherError.apply(resp);
+      if (otherError != null &&
+          !otherError.trim().isEmpty()) {
+        logger.warn(String.format("Other error occurred for region [%s]", ctxRegion));
+        // Just throw to upper layer to handle
+        throw new IllegalStateException("Received other error from TiKV:" + otherError);
```

Comment:
Unlike `region_error`, `other_error` is more unpredictable and hard to handle here, so we just throw them to upper layers to handle.